config/delegates.rb

```diff
@@ -130,10 +192,16 @@ def extra_iiif2_information_response_keys(options = {})
   def source(options = {})
     namespace, identifier = identifier_parts()
 
+    log('source switch statement, identifier: ' + identifier, 'trace')
+
     case namespace
-    when 'objectstore', 'edepot', 'beeldbank' then 'HttpSource'
-    else 'FilesystemSource'
+    when 'objectstore', 'edepot', 'beeldbank' then source = 'HttpSource'
```

Comment:
STYLE: https://github.com/rubocop-hq/ruby-style-guide#indent-when-to-case 

For readability I would put `then` on newline and use indent, but this will work as well.